hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/OidcUserManagementAutoConfiguration.java

```diff
@@ -193,7 +193,7 @@ public void onAuthenticationSuccess(final HttpServletRequest request, final Http
             final AbstractAuthenticationToken token = (AbstractAuthenticationToken) authentication;
             token.setDetails(new TenantAwareAuthenticationDetails(defaultTenant, false));
 
-            systemSecurityContext.runAsSystemAsTenant(systemManagement::getTenantMetadata, defaultTenant);
+            systemSecurityContext.runAsSystemAsTenantAsUser(systemManagement::getTenantMetadata, defaultTenant, authentication.getName());
```

Comment:
I'm not sure if this is correct here. Since this will bootstrap the initial setup from my point of view it should not be audited with the name of the user which logged in the first time. It's a pure technical one time setup thus it rather belongs to the "system" user

Reply:
Since there is another context switch in `JpaSystemManagement#createInitialTenantMetaData` back to the tenant I think `runAsSystemAsTenantAsUser` is not required here